tests/unit/sparql/buildQuery.spec.ts

```diff
@@ -1,63 +1,62 @@
 import buildQuery from '@/sparql/buildQuery';
 import PropertyValueRelation from '@/data-model/PropertyValueRelation';
+import { Condition } from '@/sparql/QueryRepresentation';
 
 describe( 'buildQuery', () => {
 
+	function getSimpleCondition( propertyId: string, value: string ): Condition {
+		const propertyValueRelation = PropertyValueRelation.Matching;
+		const simpleCondition: Condition = {
+			propertyId,
+			value,
+			datatype: 'string',
+			propertyValueRelation,
+			subclasses: false,
+			negate: false,
+		};
+
+		return JSON.parse( JSON.stringify( simpleCondition ) );
```

Comment:
This JSON.parse and JSON.stringify should not be necessary. I think this method always returns a fresh now object, not a reference to the same.